lib/seattleflu/id3c/api/routes.py

```diff
@@ -35,7 +38,16 @@ def get_metadata_v1():
 
 @api_v2.route("/shipping/augur-build-metadata", methods = ['GET'])
 @authenticated_datastore_session_required
-def get_metadata_v2(session):
+def get_metadata_v2():
```

Comment:
`@authenticated_datastore_session_required` either needs to be removed or the `session` argument needs to be added back. I think the former as the hardcoded 410 Gone response doesn't need to be behind auth, and this is what the v1 route does.

In the current state, if you do authenticate, an exception is thrown instead of a 410 Gone because the decorator tries to pass the `session` argument but the function doesn't accept it.

```
[2020-02-04 10:58:24] Connected to user=test dbname=seattleflu port=5432 sslmode=prefer
[2020-02-04 10:58:24] Exception on /v2/shipping/augur-build-metadata [GET]
Traceback (most recent call last):
  File "/home/tom/.local/share/virtualenvs/id3c-customizations-Q1pOp1dl/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/tom/.local/share/virtualenvs/id3c-customizations-Q1pOp1dl/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/tom/.local/share/virtualenvs/id3c-customizations-Q1pOp1dl/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/tom/.local/share/virtualenvs/id3c-customizations-Q1pOp1dl/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/tom/.local/share/virtualenvs/id3c-customizations-Q1pOp1dl/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/tom/.local/share/virtualenvs/id3c-customizations-Q1pOp1dl/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/tom/seattleflu/id3c/lib/id3c/api/utils/routes.py", line 37, in wrapped_route
    return route(*args, **kwargs, session = session)
TypeError: get_metadata_v2() got an unexpected keyword argument 'session'
```

Reply:
Resolving this conversation as we are no longer marking the v2 endpoint as gone at this point.